lang/LangSource/SC_LanguageConfig.cpp

```diff
@@ -219,11 +219,11 @@ void SC_LanguageConfig::freeLibraryConfig() {
     }
 }
 
-inline const bool SC_LanguageConfig::findPath(const DirVector& vec, const Path& path) {
```

Comment:
why remove inline?

Reply:
it serves no purpose here. either the compiler will inline it or it won't, and it's not going to change the linkage

Reply:
sorry for being thick, but could the `inline` have been there in the first place to avoid function call overhead?

Reply:
inline in any modern compiler is basically meaningless for determining whether a function will be inlined or not. sorry if i'm misunderstanding your question.

Reply:
that answers my question, thanks.